### PR TITLE
MISP 2.1.41 - Add Custom Object command

### PR DESCRIPTION
**Pull request by mohlcyber**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added ***misp-add-custom-object*** command to add custom objects to MISP Events.
- Updated the Docker image to: *demisto/pymisp2:2.4.184.3.87593*.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 1]
```diff
@@ -302,6 +302,28 @@ def build_generic_object(template_name: str, args: list[dict]) -> GenericObjectG
     return misp_object
 
 
+def build_custom_object(template_name: str, args: list[dict]):
+    obj = PYMISP.object_templates()
+    for ob in obj:
```
Comment (YuvHayun) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:307]: Let's try to think of a more meaningful name, maybe `for entry in obj`

File: Packs/MISP/ReleaseNotes/2_1_41.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### MISP v3
+- Added ***misp-add-custom-object*** command.
```
Comment (YuvHayun) [inline Packs/MISP/ReleaseNotes/2_1_41.md:3]:
```suggestion
- Added the command **misp-add-custom-object**, use this command to add custom objects to MISP.

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Looks good!
The failures you mentioned on slack, seems to be due to test failures, please make sure to fix and ensure they pass.

Let me know if you need any help.

File: Packs/MISP/Integrations/MISPV3/MISPV3.py (hunk 1 above)
Comment (mohlcyber) [reply Packs/MISP/Integrations/MISPV3/MISPV3.py]: Thanks for the feedback. Adjusted to entry with the latest commit.

**Comment by mohlcyber** [comment]
Based on our conversation reversed the Docker Image version. Recording is posted here: https://drive.google.com/file/d/17T8BQEVBMLcGTvI9hzUmHqhauJ86jeOK/view?usp=sharing

**Comment by YuvHayun** [comment]
Hey @mohlcyber, the video looks good and the changes as well.. moving to instance-test.

**Pull request merged**